get_images.sh

```diff
@@ -21,7 +21,8 @@ if [ ! -f "${RHCOS_IMAGE_FILENAME_OPENSTACK}" ]; then
 fi
 
 if [ ! -f ironic-python-agent.initramfs ]; then
-    curl --insecure --compressed -L https://images.rdoproject.org/master/rdo_trunk/current-tripleo/ironic-python-agent.tar | tar -xf -
+#    curl --insecure --compressed -L https://images.rdoproject.org/master/rdo_trunk/current-tripleo/ironic-python-agent.tar | tar -xf -
+    curl --insecure --compressed -L https://images.rdoproject.org/master/rdo_trunk/54c5a6de8ce5b9cfae83632a7d81000721d56071_786d88d2/ironic-python-agent.tar | tar -xf -
```

Comment:
Did /current-tripleo-rdo/.... not work? As i'd imagine this sha might be deleted at some stage

Reply:
The current-tripleo-rdo image gave me a different error but also failed.

Reply:
ok, lets merge before others start hitting the same problem, we can settle on something more permanent at a later stage.